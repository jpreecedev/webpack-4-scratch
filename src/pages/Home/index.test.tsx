import React from "react"
import { Home } from "."
import { BrowserRouter as Router } from "react-router-dom"

import { render } from "@testing-library/react"
import "@testing-library/jest-dom/extend-expect"

describe("<Home /> tests", () => {
  const renderComponent = () =>
    render(
      <Router>
        <Home />
      </Router>
    )

  it("should match snapshot", () => {
    expect(<Home />).toMatchSnapshot()
  })

  it("should render with correct as title", () => {
    const { getByTestId } = renderComponent()
    expect(getByTestId("home-title")).toHaveTextContent(
      "A bare bones React boilerplate, featuring Webpack 4, React, Jest, PostCSS and TypeScript"
    )
  })
})
