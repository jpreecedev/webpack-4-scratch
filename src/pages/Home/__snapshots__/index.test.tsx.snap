// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> tests should match snapshot 1`] = `<Home />`;
